plugins/chat/assets/javascripts/discourse/components/chat-message-actions-desktop.hbs

```diff
@@ -10,7 +10,14 @@
     }}
     data-id={{this.message.id}}
   >
-    <div class="chat-message-actions">
+    <div
+      class={{concat-class
+        "chat-message-actions"
+        (unless
+          this.messageInteractor.secondaryButtons.length "-no-more-buttons"
+        )
```

Comment:
Instead of `-no-more-buttons` -> `has-no-secondary-actions`, I understand we name it buttons in js, but this is about actions and not buttons here (chat-message-ACTIONS-container)